plugins/amongus/Game.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "Game.h"
+
+#include "mumble_plugin_utils.h"
+
+Game::Game(const procid_t id, const std::string name) : m_ok(false), m_proc(id, name) {
+	if (!m_proc.isOk()) {
+		return;
+	}
+
+	const auto modules = m_proc.modules();
+	const auto iter    = modules.find("GameAssembly.dll");
+	if (iter == modules.cend()) {
+		return;
+	}
+
+	const std::vector< uint8_t > clientPattern = { 0x74, 0x39, 0xA1, '?', '?', '?', '?', 0x8B, 0x40, 0x5C };
+	m_client = m_proc.peekPtr(m_proc.peekPtr(m_proc.findPattern(clientPattern, iter->second) + 3));
```

Comment:
Why `+ 3`? The comment about "magic numbers" applies here as well :point_up:

Reply:
I can see why it's confusing, I will write the assembly code in a comment.